test/integration/master_test.go

```diff
@@ -250,6 +251,9 @@ var deleteResp string = `
 // Job share storage.  This test can be deleted when Jobs is removed from ext/v1beta1,
 // (expected to happen in 1.4).
 func TestBatchGroupBackwardCompatibility(t *testing.T) {
+	if *testapi.Batch.GroupVersion() == v2alpha1.SchemeGroupVersion {
```

Comment:
No, because `hack/test-integration.sh` [does not have](https://github.com/kubernetes/kubernetes/blob/8a89ea605899ab8a9f8d7f2c96c98d7da3e7fa30/hack/test-integration.sh#L32) `batch/v2alph1` in `KUBE_TEST_API_VERSIONS`.